src/Viewer.js

```diff
@@ -125,11 +134,13 @@ export class Viewer {
             this.controls.maxPolarAngle = Math.PI * .75;
             this.controls.minPolarAngle = 0.1;
             this.controls.enableDamping = true;
-            this.controls.dampingFactor = 0.05;
+            this.controls.dampingFactor = 0.25;
+            this.controls.zoomToCursor = true;
```

Comment:
I'm also curious why this setting was changed.

Reply:
Same thing as previous, a little more control when trying to get specific angles when editing. Also this one should be inside the Editor class though.

Reply:
Could we make a toggle in the Editor UI for this?

Reply:
Absolutely